ArticleTemplates/assets/scss/garnett-modules/content/_meta.scss

```diff
@@ -103,7 +103,7 @@
         margin-left: base-px(.5);
         border-left: 1px solid color(brightness-86);
         margin-bottom: base-px(.5);
-        margin-top: base-px(-0.5);
+        margin-top: base-px(-.5);
```

Comment:
This is an unrelated scss validation fix that was flagged when I ran `yarn validate` locally.